_collections/_news/2024-09-22-css-welcome-week-newsletter-2024.md

```diff
@@ -0,0 +1,84 @@
+**Welcome CSS members, to a new year of UoB!**
```

Comment:
Since this is markdown, you can probably use headers here, e.g. `# Welcome CSS members`